Gopkg.toml

```diff
@@ -96,7 +96,7 @@ required = [
 
 [[override]]
   name = "github.com/giantswarm/k8scloudconfig"
-  version = "~0.1.0"
+  branch = "master"
```

Comment:
I think this should still be `~0.1.0` until `aws-operator` switches to go modules. This probably works, but transitive dependencies might not be correct.